CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## To be released
 - [Android] Updated to be compatible with Android Cordova v5.1.1
 - [iOS] Include pushclient module in iOS build
+- Add support for iOS universal deeplinking
```

Comment:
Nitpick... What about prefixing the entry with the platform like other entries above it?  
```
- [iOS] Add support for universal links
```